index.html

```diff
@@ -0,0 +1,215 @@
+                                <li>MyPage</li>
+                                <li>Logout</li>
+                            </ul>
+                        </div>
+                        <div class="menu">
+
+                            <img
+                                src="/first-capstone-project/assets/icons/nav-icon.png"
+                                class="mobile"
+                                id="menu" onclick="menuOverlay()" />
+                            <h2 id="logo"><a href="index.html">Amanna Trust</a>
+                            </h2>
+
+                            <img
+                                src="/first-capstone-project/assets/icons/close-icon.png"
+                                class="mobile"
+                                id="menu-close" />
+
+                            <ul class="nav-links" id="nav-links">
+
+                                <li class="nav-link active"><a
+                                        href="/first-capstone-project/index.html">Home</a></li>
+                                <li class="nav-link"><a
+                                        href="/first-capstone-project/pages/about.html">About</a></li>
+                                <li class="nav-link">Program</li>
+                                <li class="nav-link">join</li>
+                                <li class="nav-link">Sponsor</li>
+                                <li class="nav-link">News</li>
+                            </ul>
+                            <div class="campaign-donate mobile desktop">
+                                <p class="campaign-donate-text">Donate</p>
+                            </div>
+                            <div class="close">
+                                <img src="assets/icons/close-icon.png"
+                                    class="mobile"
+                                    id="menu-close" />
+                            </div>
+                        </div>
+
+                    </div>
+
+                </nav>
+            </header>
+            <section class="hero">
+                <div class="headline">
+                    <p>Your help goes a long way!</p>
+                    <h1 class="title-h1">Help us reach needy families and
+                        empower them get
+                        access to education. </h1>
+                </div>
+
+                <div class="card">
+                    <p class="headline-text">We aim to shape possibilities of
+                        under privileged
+                        children by providing a stepping stone to acquiring
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
+
+        </div>
+        <section class="values">
+            <h3>Our Values</h3>
+            <hr>
```

Comment:
kindly increase the font weight and text size of the title and reduce the width of the devider to make it look like the design 

![Capture d’écran du 2023-08-02 00-22-30](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/6163dec9-85ec-4e6a-9fb9-ef2b3bbb6451)
![Capture d’écran du 2023-08-02 00-22-10](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/69dc6d3f-2348-4d80-ae6b-8c7f17f9d235)